### ISSUE TITLE
`lint/useImportType` and `lint/useExportType` code fixes wrongly handle multiline statements

### ISSUE THREAD

**Issue by revosw** [biomejs/biome#2041]
### Environment information

```bash
CLI:
  Version:                      1.6.0
  Color support:                true

Platform:
  CPU Architecture:             x86_64
  OS:                           windows

Environment:
  BIOME_LOG_DIR:                unset
  NO_COLOR:                     unset
  TERM:                         unset
  JS_RUNTIME_VERSION:           "v21.6.1"
  JS_RUNTIME_NAME:              "node"
  NODE_PACKAGE_MANAGER:         "pnpm/8.1.1"

Biome Configuration:
  Status:                       Loaded successfully
  Formatter disabled:           false
  Linter disabled:              false
  Organize imports disabled:    false
  VCS disabled:                 true

Linter:
  Recommended:                  true
  All:                          false
  Rules:
Workspace:
  Open Documents:               0
```

### Rule name

useImportType

### Playground link

https://github.com/revosw/biome-useImportType-repro

### Expected result

Expected `export`:
```ts
export {
	a,
	type AAAAAAAAAAAAAA,
	type BBBBBBBBBBBBBB,
	type CCCCCCCCCCCCCC,
	type DDDDDDDDDDDDDD,
	type EEEEEEEEEEEEEE,
};
```
Actual `export`:
```ts
export {
	a,
	type
	AAAAAAAAAAAAAA,
	type
	BBBBBBBBBBBBBB,
	type
	CCCCCCCCCCCCCC,
	type
	DDDDDDDDDDDDDD,
	type
	EEEEEEEEEEEEEE,
};
```
Expected `import`:
```ts
import {
	a,
	type AAAAAAAAAAAAAA,
	type BBBBBBBBBBBBBB,
	type CCCCCCCCCCCCCC,
	type DDDDDDDDDDDDDD,
	type EEEEEEEEEEEEEE,
} from "./exportedstuff.ts";
```
Actual `import`:
```ts
import {
	a,
	type
	AAAAAAAAAAAAAA,
	type
	BBBBBBBBBBBBBB,
	type
	CCCCCCCCCCCCCC,
	type
	DDDDDDDDDDDDDD,
	type
	EEEEEEEEEEEEEE,
} from "./exportedstuff.ts";
```

### Code of Conduct

- [X] I agree to follow Biome's Code of Conduct
